sdks/java/core/src/main/java/org/apache/beam/sdk/util/PCollectionViews.java

```diff
@@ -165,8 +165,8 @@ public T getDefaultValue() {
             throw new RuntimeException("Unexpected IOException: ", e);
           }
         }
+        return defaultValue;
```

Comment:
The FindBugs issue this is addressing is "inconsistent synchronization" because the write is inside the sync block and read was outside. Since the synchronization is used for one-time lazy init, there's really no danger of reading a different value, so this change is only to make the tooling happy.